src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -137,7 +125,7 @@ export const FoxEthProvider = ({ children }: FoxEthProviderProps) => {
     if (farmingAccountId && transaction && transaction.status !== TxStatus.Pending) {
       if (transaction.status === TxStatus.Confirmed) {
         moduleLogger.info('Refetching fox lp/farming opportunities')
```

Comment:
wanna update this comment too?

Reply:
https://github.com/shapeshift/web/pull/3585/commits/8a6d5887bc5f7e96874198e2f3431672af2565bf